src/Nethermind/Nethermind.AccountAbstraction/Source/PaymasterThrottler.cs

```diff
@@ -49,19 +49,16 @@ public class PaymasterThrottler : IPaymasterThrottler
                 )
             );
 
-        public PaymasterThrottler()
-        {
-            _opsSeen = new Dictionary<Address, uint>();
-            _opsIncluded = new Dictionary<Address, uint>();
+        public PaymasterThrottler() : this(true) { }
 
-            SetupAndStartTimer();
-        }
+        public PaymasterThrottler(bool isBundler)
+            : this(isBundler, new Dictionary<Address, uint>(), new Dictionary<Address, uint>()) { }
 
-        public PaymasterThrottler(
-            IDictionary<Address, uint> opsSeen,
-            IDictionary<Address, uint> opsIncluded
-        )
+        public PaymasterThrottler(bool isBundler,
+                IDictionary<Address, uint> opsSeen, IDictionary<Address, uint> opsIncluded)
         {
+            MinInclusionRateDenominator = isBundler ? (uint)10 : (uint)100;
```

Comment:
lets call this isMiner instead to avoid confusion with flashbots

Reply:
OK, that's fair

Reply:
Done